gradle.properties

```diff
@@ -11,4 +11,4 @@ mavenPassword=YourPassword
 # - here-naksha-lib-core/NakshaVersion (static property: latest)
 # - here-naksha-lib-psql/resources/naksha_plpgsql.sql (method: naksha_version)
 # - here-naksha-app-service/src/main/resources/swagger/openapi.yaml (info.version property)
-version=2.1.1
+version=2.1.2-SNAPSHOT
```

Comment:
With automated pipeline, once we tag 2.1.2, the Gitlab pipeline will automatically build and publish to HERE artifactory. Hence, we can (should) directly use the final version here. i.e. 2.1.2